### PR TITLE
Removed duplicates from OCRFixReplaceLists

### PR DESCRIPTION
**Pull request by xylographe**

### CODE REVIEW

**Comment by xylographe** [comment]
Re-based

File: Dictionaries/dan_OCRFixReplaceList.xml [hunk 1]
```diff
@@ -119,11 +118,9 @@
     <Word from="ﬁnder" to="finder" />
     <Word from="ﬁndes" to="findes" />
     <Word from="irettesaeﬁelse" to="irettesættelse" />
-    <Word from="sy’g" to="syg" />
```
Comment (niksedk) [inline Dictionaries/dan_OCRFixReplaceList.xml]: is this a duplicate?

Comment (xylographe) [inline Dictionaries/dan_OCRFixReplaceList.xml]: Lines [77](https://github.com/SubtitleEdit/subtitleedit/blob/master/Dictionaries/dan_OCRFixReplaceList.xml#L77) and [122](https://github.com/SubtitleEdit/subtitleedit/blob/master/Dictionaries/dan_OCRFixReplaceList.xml#L122).

These diffs are generated by a powershell-script using a System.Collections.Generic.Dictionary with ordinal string comparison—except for the typo fix in `hrv_OCRFixReplaceList.xml`, which I noticed while checking the result in a side-by-side diff. :)

Comment (niksedk) [inline Dictionaries/dan_OCRFixReplaceList.xml]: cool :)

**Pull request merged**